test/WebJobs.Script.Tests/LinuxContainerInitializationServiceTests.cs

```diff
@@ -0,0 +1,149 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Can you put this test into a ContainerManagement folder corresponding to the new namespace you added?

Reply:
Done.